### PR TITLE
Fix: CHANGELOG - Add missing entry for 559

### PR DESCRIPTION
**Pull request by oxedions**
I forgot to add the CHANGELOG entry for PR #559

### CODE REVIEW

**Review by hmeAtos — COMMENTED** [review]
"kernel_config: prevent crash if variable is None" 
What is this change ?

Regards

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
   - diskless:
     - fix issues with dnf command in the livenet module (#528)
     - fix issues preventing access to nodes booting a livenet image (#525)
+  - kernel_config: prevent crash if variable is None (#559)
```
Comment (btravouillon) [inline CHANGELOG.md:14]: I would add the variable name: ...if variable ep_kernel_parameters is undefined.

Comment (oxedions) [reply CHANGELOG.md]: Should be ok now.

My deepest apologies for this very long delay, I was totally over loaded.

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Comment by oxedions** [comment]
Thanks a lot ! 😊

**Pull request merged**